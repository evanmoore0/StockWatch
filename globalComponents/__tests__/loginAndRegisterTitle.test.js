
import React from "react";
import renderer from 'react-test-renderer'
import LoginAndRegisterTitle from "../loginAndRegisterTitle";

describe('<LoginAndRegisterTitle/>', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<LoginAndRegisterTitle/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
})
