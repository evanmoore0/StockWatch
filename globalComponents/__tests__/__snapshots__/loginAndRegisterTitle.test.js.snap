// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginAndRegisterTitle/> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "flex-start",
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 81,
      }
    }
  />
</View>
`;
